include/iota/types/trinary.hpp

```diff
@@ -64,6 +64,10 @@ bool isValidHash(const Trytes& s);
  **/
 bool isArrayOfHashes(const std::vector<Trytes>& hashes);
 
+Trytes      charToTrytes(const char c);
```

Comment:
can we rename that to byteToTryte / bytesToTrytes?
That would be consistent with bytesToTrits, or the meaning of the functions is different?

Can you also add comments

Reply:
Done